swift_browser_ui_frontend/src/components/CFooter.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div id="c-footer">
+    <nav class="navbar">
+    <div class="navbar-menu is-active">
+        <div class="navbar-start" >
+          <div class="col-3">
+            <div class="row navbar-item " >
+              <span class="largetext">
+              {{ $t("message.footerMenu.groupandprogram") }}
+            </span>
+            </div>
+            <div class="row navbar-item smalltext">
+              <a class="linktext"
+              href="https://csc.fi"
+              :alt="$t('message.footerMenu.csc')"
+            >{{
+              $t("message.footerMenu.csc")
+            }}</a>
```

Comment:
If we could have the codes in their correct lines and indentation, for example this one, it would be nicer to read :)